docs/src/modules/components/AppSearch.js

```diff
@@ -143,6 +144,9 @@ export default function AppSearch() {
 
   React.useEffect(() => {
     if (desktop) {
+      // In non-SSR languages, fall back to English.
+      const facetFilterLanguage = LANGUAGES_SSR.includes(userLanguage) ? `language:${userLanguage}` : `language:en`
```

Comment:
We haven't dropped IE 11 yet. When we do we look at these manual polyfills individually. In the meantime: 
```suggestion
      const facetFilterLanguage = LANGUAGES_SSR.indexOf(userLanguage) !== -1 ? `language:${userLanguage}` : `language:en`
```